drivers/epson/projector/esc_vp21.cr

```diff
@@ -0,0 +1,274 @@
+require "digest/md5"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/switchable"
+
+module Epson; end
+
+module Epson::Projector; end
+
+# Documentation: documentation link
+
+class Epson::Projector::EscVp21 < PlaceOS::Driver
+  include Interface::Powerable
+  include Interface::Muteable
+
+  enum Inputs
+    HDMI
+    HDBaseT
+  end
+
+  # Discovery Information
+  tcp_port 1024
+  descriptive_name "Epson Projectors"
+  generic_name :Display
+
+  def on_load
+    transport.tokenizer = Tokenizer.new("\r\n")
+
+    self[:volume_min] = 0
+    self[:volume_max] = 255
+
+    self[:power] = 0
+    self[:stable_state] = true
+
+    self[:type] = :projector
+  end
+
+  def on_update
+  end
+
+  def connected
+    # Have to init comms
+    send("ESC/VP.net\x10\x03\x00\x00\x00\x00")
+    do_poll
+    schedule.every(52.seconds) { do_poll }
```

Comment:
In cases where you want to create a repeating schedule and have it run straight away too you can use `schedule.every(52.seconds, true) { ... }`